Mathlib/Data/Set/Lattice.lean

```diff
@@ -728,6 +731,9 @@ theorem biInter_subset_of_mem {s : Set α} {t : α → Set β} {x : α} (xs : x
     ⋂ x ∈ s, t x ⊆ t x :=
   iInter₂_subset x xs
 
+lemma biInter_subset_biUnion {ι : Type*} {s : Set ι} (hs : s.Nonempty) {f : ι → Set α} :
```

Comment:
I'd say you should use `{t : α → Set β}` for consistency with the surrounding lemmas here